src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java

```diff
@@ -208,8 +208,15 @@ public void scan(HttpMessage msg, String param, String value) {
                     msg = getNewMsg();
                     setParameter(msg, param, prefix + target);
 
-                    //send the modified request, and see what we get back
-                    sendAndReceive(msg);
+                  //send the modified request, and see what we get back
+                    try {
+                    	sendAndReceive(msg);
+                    } catch (IllegalStateException ex) {

```

Comment:
Good catch, the later [3 lines](https://github.com/zapbot/zap-mgmt-scripts/blob/5332d8365a0546b2ff6a3e6782627e1e259cf6c7/reports/wavsep-1.5-2.4.3-RB-M-M.logs.txt#L1575-L1577) do appear to be unknown hosts. Will fix that too.